accepted/00076-state-channels-visibility.md

```diff
@@ -0,0 +1,218 @@
+![Configuration Channels Detail Systems AFTER](images/state-channels-visibility/Configuration_Channels_Details_Systems_AFTER.png)
+
+
+The list could look like:
+```
+|----------------+-----------------|
+| System name    | Inherited from  |
+|----------------+-----------------|
+| test01.suse.de | org1, my group1 |
+| test02.suse.de | my group2       |
+|----------------+-----------------|
+```
+
+
+### `Configuration > Channels > Details > System Groups`
+This page reproduces the same behavior of the `Configuration > Channels > Details > Systems` tab, but for System Groups. Being able to filter by System Group would be a nice but not strictly required addition.
+
+This **new** page is pointed by:
+- the `Channel information` box link of the system groups counting in the `Configuration > Channels > Details > Overview` page: the link will land on the new page at the `Subscribed System Groups` tab
+
+### `Configuration > Channels > System Groups`
+This page represents a complete list of pairs value `State Channel - System Group`: every value is a link to the specific entity detail (channel or system group). Filters for both fields should be available in order to look for a certain State Channel or a System Group or both.
+
+This **new** page is pointed by:
+- the `Configuration summary` box link of the system groups counting in the `Configuration > Overview` page: the link will land on the new page where all the pairs `State Channel - System Group` are listed
+- the `Configuration > Channels` list, for every channel entry, from the `Subscribed System Groups` counting link value
+
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+## Why should we **not** do this?
+
+No real reason for not enhancing the feature adding the missing visibility.
+
+The concrete drawback is: the solution is adding information to a tech-debt affected architecture, spreading the same information in many places (see how many pages/screenshots). Maintaining should not be complicated, but future refactoring of the feature workflow could be more difficult.
+
+# Alternatives
+[alternatives]: #alternatives
+
+## What other designs/options have been considered?
+
+Without investigating deeply, the alternative would be redesigning the feature completely from both the design and technical perspectives: the technical refactoring should get rid of the struts stack and redefine the concept of Configuration Channel and State Channel used by traditional or salt minion clients. By addressing the technical part, also the workflow could be re-drawn, dropping some pages and instead of adding separated tabs of tables it would be preferrable to have unified modern lists of filterable entries. The drawback of this alternative is that it would take much longer (from 3 to 5 times roughly) than the proposed official solution.
+
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+## What are the unknowns?
+1. Salt States coming from Formulas are not yet considered as part of this RFC
```

Comment:
At this point, this is not an unknown, it should rather be presented as a known limitation of the proposed approach.